Documentation/api_rfc/v2_acl_api.md

```diff
@@ -0,0 +1,253 @@
+# v2 Auth and ACL
```

Comment:
how about dropping the ACL terminology in the doc and mentioning it is a role-base access control in a overview section?